packages/components/tree/src/tree-node.vue

```diff
@@ -184,6 +186,23 @@ export default defineComponent({
       return getNodeKeyUtil(tree.props.nodeKey, node.data)
     }
 
+    const getNodeClass = (node: Node) => {
+      const nodeClassFunc = props.props.class
+      let className = nodeClassFunc
+      if (nodeClassFunc instanceof Function) {
+        const { data } = node
+        className = nodeClassFunc(data, node)
+      }
+      if (!className) {
+        return {}
+      }
+      if (typeof className === 'string') {
```

Comment:
Similar here you can use exported helpers to determine whether this is a string type.

```ts
import { isString } from '@element-plus/utils/util'
```
or

```ts
import { isString } from '@vue/shared'
```

They are both acceptable.